posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+
+This blog post is the first in our series on using our
+`napari-cosmx` plugin. Like other items in our [CosMx Analysis Scratch Space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html),
+the `napari-cosmx` plugin is experimental so the usual [caveats]() and [license](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/license.html) applies.
+Future posts will do deep-dives into _using_ the `napari-cosmx` plugin. This post will
+show you how to:
+
+- @sec-installing Install napari and the `napari-cosmx` plugin 
+- @sec-napari-ready-files Describe what our "napari-ready" data format is 
+- @sec-loading-slides How to load a napari-ready slide 
+- @sec-stitching How to create a napari-ready slide from AtoMx raw data export 
+- @sec-adding-metadata Minimum example of how to view cell-level metadata 
+
+::: {.callout-note}
+`napari-cosmx` is actively and continuously under development in the RnD groups at
+NanoString. We do not (yet) have the source code for `napari-cosmx` opened-sourced.
+Please be aware that there may be bugs and that it has not gone through the regular
+level of quality and testing. Our goal here is to bring the capabilities of napari
+to CosMx users as fast as possible.
+:::
+
+## What is Napari?
+
+Stepping back for a second. What exacty _is_ Napari. [Napari](https://napari.org/){target="_blank"}
+is an [open-source](https://github.com/napari/napari){target="_blank"} Python project that runs a 
+Qt-based desktop GUI for interactive visualization of scientific images. 
+The application has layers of different types, similar to what you might find
+in application like Photoshop or Procreate. 
+
+## What is the `napari-cosmx` plugin?
+
+Tissue morphology layers generated by CosMx SMI are displayed as image layers 
+in napari, and standard controls are available such as opacity, gamma, and contrast
+limits. Protein results are also viewed as an image layer in napari. For CosMx 
+RNA experiments, the detected transcripts are viewed as a points layer in napari.
+Cell segmentation results are displayed as an image layer of the cell boundaries.
+The cell shapes can also be colored by metadata such as cell type. @fig-example-gif
+shows some examples of these in an animated format. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-example-gif"
+#| fig-cap: "Example animation made with `napari-cosmx` showing ligand-receoptor interactions in a healthy prostate sampele. Cell types fill in the cell boundaries. When cells are transparent, one can see more easily see the spatial location of _S100A8_, _S100A9_ ligand RNA transcripts with the _TLR4_ receptor transcript. While animations are certainly not needed for all purposes, this one highlights that the plugin can color RNA transcripts (points layer), cell-level metadata like cell types, and cell boundaries. It can also visualize IF (not shown here; see @fig-pancreas for example of IF staining)." 
```

Comment:
receoptor -> receptor
sampele -> sample
'_S100A8_, _S100A9_' -> '_S100A8_ and _S100A9_'
'visualize IF' -> 'visualize immunofluorescence (IF)'
What is the layer type for cell types and cell boundaries? This might be a good place to add that, after noting that transcripts are in a points layer. Alternatively, could omit the note that transcripts are points.

Reply:
Done: 787b3c0dab0afb0689625ac6107ffa0428c97cfa